install.sh

```diff
@@ -0,0 +1,459 @@
+#!/bin/bash
+set -u
+
+# This script installs to /usr/local only. To install elsewhere (which is
+# unsupported) you can untar https://github.com/Homebrew/brew/tarball/master
+# anywhere you like.
+HOMEBREW_PREFIX="/usr/local"
```

Comment:
Bash has *a lot* of pitfalls and places where things can go wrong if we’re distracted.

I’d feel safer if every variable would be `readonly` when possible (i.e. if it’s not you know it will change) and used single quotes unless double quotes are necessary.